.github/workflows/build.yaml

```diff
@@ -0,0 +1,49 @@
+name: Continuous Integration
+
+on:
+    pull_request: ~
+    push: ~
+
+jobs:
+    tests:
+        runs-on: ubuntu-20.04
+        strategy:
+            max-parallel: 1
+            matrix:
+                php:
+                    - '7.2'
+                    - '7.3'
+                    - '7.4'
+                    - '8.0'
+                include:
+                    - description: 'Symfony 3.4'
+                      php: '7.2'
+                      composer_option: '--prefer-lowest'
+                    - description: 'Symfony 4.4'
+                      php: '7.3'
+                      symfony: 4.4.*
+                    - description: 'Symfony 5.1'
+                      php: '7.4'
+                      symfony: 5.1.*
```

Comment:
should be removed. Jobs forcing a precise version of Symfony should be used only for LTS versions IMO, so that this needs to be maintained every 2 years rather than every 6 months when versions become EOL.